@xen-orchestra/lite/src/components/ui/UiTable.vue

```diff
@@ -1,45 +1,55 @@
 <template>
-  <table class="ui-table">
-    <thead>
-      <tr class="header-row">
-        <slot name="header" />
-      </tr>
-    </thead>
-    <tbody class="body">
-      <slot />
-    </tbody>
+  <table :class="{ 'vertical-border': verticalBorder }" class="ui-table">
+    <slot />
   </table>
 </template>
 
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+defineProps<{
+  verticalBorder?: boolean;
+}>();
+</script>
 
 <style lang="postcss" scoped>
 .ui-table {
+  width: 100%;
   border-spacing: 0;
-}
+  background-color: var(--background-color-primary);
+  font-weight: 400;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+  color: var(--color-blue-scale-200);
 
-:deep(th),
-:deep(td) {
-  padding: 1rem;
-  border-top: 1px solid lightgrey;
-  border-right: 1px solid lightgrey;
-  text-align: left;
+  :deep(th),
+  :deep(td) {
+    padding: 1rem;
+    border-top: 1px solid var(--color-blue-scale-400);
+    text-align: left;
```

Comment:
It won't always be valid. For example, we should display the numbers on the right.

Reply:
I don't understand what you mean.

Whatever the default alignment may be, if a different one is required, you'll need to add a CSS class/rule.